modules/config-idp.adoc

```diff
@@ -10,6 +10,11 @@ After you have installed {product-title}, you must configure your cluster to use
 
 You can configure different identity provider types for your {product-title} cluster. Supported types include GitHub, GitHub Enterprise, GitLab, Google, LDAP, OpenID Connect, and HTPasswd identity providers.
 
+[IMPORTANT]
+====
+The `htpasswd` identity provider option is included only to enable the creation of a single, static administration user. `htpasswd` is not supported as a general use identity provider for {product-title}.
```

Comment:
I think `general-use` is a compound adjective here, but it seems to be one that basically no one hyphenates. Also, is there a not-too-onerous way to get the `htpasswd` behind some other word in the second sentence so it doesn't start the sentence?
```suggestion
The `htpasswd` identity provider option is included only to enable the creation of a single, static administration user. `htpasswd` is not supported as a general-use identity provider for {product-title}.
```

Reply:
@jeana-redhat thank you for these suggestions. I have subsequently applied "general-use" throughout the PR.

> Also, is there a not-too-onerous way to get the htpasswd behind some other word in the second sentence so it doesn't start the sentence?

I have updated each note to use "HTPasswd" rather than `htpasswd`, as per the suggestion from @bysnupy. Subsequently, the sentence no longer starts with a code-formatted word, which I am assuming resolves this point.